stanford_profile.post_update.php

```diff
@@ -46,3 +49,14 @@ function stanford_profile_post_update_8014() {
   ]);
 
 }
+
+/**
+ * Update the new config_ignore settings prior to config import.
+ */
+function stanford_profile_post_update_8015() {
```

Comment:
if the blocks get prefixed with `stanford_basic` this update won't be necessary.